src/Cache/FileCache.php

```diff
@@ -79,19 +79,19 @@ protected function file(string $key): string
 				case '/':
 					break;
 
-				// backslashes get their own marker in the path
-				// to differentiate the cache key from one with forward slashes
+					// backslashes get their own marker in the path
+					// to differentiate the cache key from one with forward slashes
```

Comment:
Could you please revert the unrelated coding style changes in `Cache\FileCache`, `Cms\Email`, `Database\Sql` and `Toolkit\Html`?

Reply:
@lukasbestle I get the same changes since a while locally from cs fixer. But somehow the CI workflow doesn't want them? Though agreed they shouldn't be in this PR

Reply:
Reverted! But yup, that behavior from `php-cs-fixer` is pretty annoying, have to manually run it once for all files I've touched 😅